builtin/game/item.lua

```diff
@@ -139,6 +139,19 @@ function core.dir_to_wallmounted(dir)
 	end
 end
 
+function core.wallmounted_to_dir(wallmounted)
+	-- table of dirs in wallmounted order
+	return ({[0]={x=0, y=1, z=0},
+					{x=1, y=-1, z=0},
+					{x=1, y=0, z=0},
+					{x=-1, y=0, z=0},
+					{x=0, y=0, z=1},
+					{x=0, y=0, z=-1}})
+
+					--indexed into by the wallmounted in question
+					[wallmounted]
+end
+
```

Comment:
I don't really know about lua, though. But I think in most other languages code and allocated data are stored as a different representation.

If the table were already stored in memory using the same representation then there would be no speed advantage for placing it outside, since it would have been created already, wouldn't it?